Enterprise-CMCS/macpro-onemac
# Oy2-30166 SubSub Prevent CMS Action

Story: https://qmacbis.atlassian.net/browse/OY2-30166
Endpoint: See github-actions bot comment

### Details

Under some statuses ("Pending - Approval" or "Pending - Canceled" ) CMS users were able to see the submit subsequent documets action. CMS should never have the submit subsequent documents action.

### Changes

- fix bracing issue in actiondelegate.js

### Test Plan

1. Login as a CMS user (such as onemaccpoc6@gmail.com, onemacsrt52@gmail.com, and sysadmin@nightwatch.test)
2. Find a package in any state starting with "Pending - Approval" or "Pending - Canceled" and open it
3. Verify the action for subsequent document upload is not available
